rave-compiler/src/main/java/com/uber/rave/compiler/CompilerUtils.java

```diff
@@ -47,11 +47,11 @@
 import javax.lang.model.util.Types;
 
 /**
- * Utitlies class for annotations.
+ * Utilities class for annotations.
  */
 final class CompilerUtils {
 
-    private static final List<Class<? extends Annotation>> SUPPORTED_ANNOTATIONS = ImmutableList.of(
+    private static final List<Class<? extends Annotation>> SUPPORT_ANNOTATIONS = ImmutableList.of(
             NonNull.class,
             Nullable.class,
```

Comment:
Gotcha. So if we make it `compile` and get consumers to specify the `maven.google.com` in their projects this should work - right?